### ISSUE TITLE
picklist - listbox violations

### ISSUE THREAD

**Issue by srujana10499** [primefaces/primeng#18941]
### Describe the bug

The p-picklist component has an accessibility violation when running the IBM Equal Accessibility Checker: "Interactive component with ARIA role 'tree' does not have a programmatically associated name"

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz-starters-bwt9ucm2.stackblitz.io

### Environment

Stackblitz

### Angular version

20.0.4

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Install the [IBM Equal Accessibility Checker](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp)
2. Open the stackblitz
3. Run the accessibility checker
4. Filter on violations only and note the error: 'Interactive component with ARIA role 'tree' does not have a programmatically associated name'

### Expected behavior

No accessibility violations are present for the p-picklist and p-listbox component
